generator/integration_tests/golden/internal/golden_thing_admin_auth_decorator.h

```diff
@@ -0,0 +1,148 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Generated by the Codegen C++ plugin.
+// If you make any local changes, they will be lost.
+// source: generator/integration_tests/test.proto
+
+#ifndef GOOGLE_CLOUD_CPP_GENERATOR_INTEGRATION_TESTS_GOLDEN_INTERNAL_GOLDEN_THING_ADMIN_AUTH_DECORATOR_H
+#define GOOGLE_CLOUD_CPP_GENERATOR_INTEGRATION_TESTS_GOLDEN_INTERNAL_GOLDEN_THING_ADMIN_AUTH_DECORATOR_H
+
+#include "generator/integration_tests/golden/internal/golden_thing_admin_stub.h"
+#include "google/cloud/internal/unified_grpc_credentials.h"
+#include "google/cloud/version.h"
+#include <google/longrunning/operations.grpc.pb.h>
+#include <memory>
+#include <set>
+#include <string>
+
+namespace google {
+namespace cloud {
+namespace golden_internal {
+inline namespace GOOGLE_CLOUD_CPP_GENERATED_NS {
+
+
+class GoldenThingAdminAuth : public GoldenThingAdminStub {
+ public:
+  ~GoldenThingAdminAuth() override = default;
+  GoldenThingAdminAuth(
+      std::shared_ptr<google::cloud::internal::GrpcAuthenticationStrategy> auth,
+      std::shared_ptr<GoldenThingAdminStub> child);
+
+
```

Comment:
Nit: Did we intend to generate multiple newlines?

Reply:
I am going to punt on these to #5821, in general the code generated in `google/cloud/*` gets the `clang-format` treatment. The code in `generator/integration_tests/*` does not (yet).